src/gui/rss/rss_imp.cpp

```diff
@@ -647,20 +659,26 @@ void RSSImp::updateItemInfos(QTreeWidgetItem *item)
 void RSSImp::updateFeedIcon(const QString &url, const QString &iconPath)
 {
     QTreeWidgetItem *item = m_feedList->getTreeItemFromUrl(url);
-    item->setData(0, Qt::DecorationRole, QVariant(QIcon(iconPath)));
+
+    if (item != 0)
```

Comment:
Because I'm still trying to get into the right mindset ... day job is currently mainly Java, which has syntax close enough to C++ that it's messing with switching my mindset (I also commonly use Python in which the above idiom would be natural).